ui/app/pages/home/home.component.js

```diff
@@ -74,10 +76,12 @@ export default class Home extends PureComponent {
     originOfCurrentTab: PropTypes.string,
     disableWeb3ShimUsageAlert: PropTypes.func.isRequired,
     pendingConfirmations: PropTypes.arrayOf(PropTypes.object).isRequired,
+    infuraBlocked: PropTypes.bool.isRequired,
   };
 
   state = {
     mounted: false,
+    canShowBlockageNotification: true,
```

Comment:
This might be pretty annoying for users, as it'll be restored every time they navigate back to the home page. e.g. "Home => Settings => Home" will force them to dismiss it again.

But nonetheless, I like this solution for now. It's simple and pragmatic. We can improve the dismissal logic later if we feel the need. Given that there's not a lot users can do in this state, it might not be super important to improve this.

Reply:
My exact thoughts here as well, I think when we approach improving network connectivity UX as a whole, this should definitely be a point for refactoring